common/cpp/src/google_smart_card_common/logging/logging.h

```diff
@@ -155,7 +155,7 @@ std::string MakeNotreachedHitMessage(const std::string& file,
 #define GOOGLE_SMART_CARD_NOTREACHED                                \
   do {                                                              \
     GOOGLE_SMART_CARD_LOG_FATAL                                     \
-        << ::google_smart_card::internal::MakeNotreachedHitMessage( \
+        << ::google_smart_card::internal::MakeNotreachedMessage(    \
```

Comment:
nit: Check whether clang-format reformats this code by removing extra spaces till the backslash.